components/cam/src/physics/cam/micro_p3_interface.F90

```diff
@@ -155,6 +156,10 @@ subroutine micro_p3_readnl(nlfile)
      write(iulog,'(A30,1x,L)')    'micro_aerosolactivation: ', micro_aerosolactivation
      write(iulog,'(A30,1x,L)')    'micro_subgrid_cloud: ',     micro_subgrid_cloud
      write(iulog,'(A30,1x,L)')    'micro_tend_output: ',       micro_tend_output
+     write(iulog, *)              'p3_QcAutoCon_Expon',        p3_QcAutoCon_Expon
+     write(iulog, *)              'p3_CldImmFrz_Expon',        p3_CldImmFrz_Expon
+     write(iulog, *)              'p3_QcAccret_Fact',          p3_QcAccret_Fact
+     write(iulog, *)              'p3_QcAccret_Expon',         p3_QcAccret_Expon
```

Comment:
Again, just an aesthetic change and non-essential.  But if it isn't too much trouble could we format this log-output so that it reads well.  In this case I'm guessing replacing * with '(A30,1x,8.4e)' would probably do the trick.